jobs/build.jpl

```diff
@@ -79,21 +79,26 @@ node("docker" && params.NODE_LABEL) {
     K8S ctx:   ${k8s_context}""")
 
     docker_image = "${params.DOCKER_BASE}k8s:kernelci"
-    j.dockerPullWithRetry(docker_image).inside("-v $HOME/.kube:/.kube -v $HOME/.config/gcloud:/.config/gcloud -v $HOME/.azure:/.azure") {
+    j.dockerPullWithRetry(docker_image).inside("-v $HOME/.kube:/.kube-host:ro -v $HOME/.config/gcloud:/.config/gcloud -v $HOME/.azure:/.azure") {
         build_env_docker_image = j.dockerImageName(
             params.BUILD_ENVIRONMENT, params.ARCH
         )
+        env.KUBECONFIG="/tmp/.kube/config"
 
         stage("Init") {
-            /* Remove any leftover temporary files */
+            /* Remove any leftover temporary files 
             sh(script: "rm -rf *")
+            */
+            /* Copy host kubernetes configs */
+            sh(script: "mkdir /tmp/.kube")
+            sh(script: "cp -r /.kube-host/* /tmp/.kube")
             /* list clusters to init/refresh auth credentials */
             print("K8S: Google Cloud clusters available:")
             sh(script: "gcloud container clusters list || exit 0")
             print("K8S: Azure clusters available:")
             sh(script: "az aks list -o table || exit 0")
             print("K8S context: ${k8s_context}.  Current nodes:")
-            sh(script: "kubectl --context ${k8s_context} get nodes")
+            sh(script: "kubectl --context ${k8s_context} get nodes || exit 0")
```

Comment:
It's worth putting a comment why we're silencing the error

Reply:
done